src/metrics.rs

```diff
@@ -45,4 +45,14 @@ impl Metrics {
         let hits = self.bloom_filter_hits.load(Relaxed) as f64;
         hits / queries
     }
+
+    /// Number of bloom filter queries performed.
+    pub fn bloom_filter_queries(&self) -> usize {
```

Comment:
Also, I named this wrong in `bloom_filter_efficiency`, but it should be called `filter_*`, there may be other types than Bloom filters.

Reply:
removed as many references of "bloom"  as possible to make it generic